packages/subproviders/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@0xproject/subproviders",
+  "version": "0.0.1",
+  "main": "lib/src/index.js",
+  "types": "lib/src/index.d.ts",
+  "license": "Apache-2.0",
+  "scripts": {
+    "prebuild": "npm run clean",
```

Comment:
does prebuild get called with `lerna run build` ?

Reply:
No, removed this. It's legacy from Dekz's repo.